P/PooledArrays/Compat.toml

```diff
@@ -4,5 +4,5 @@ julia = "0.5-0"
 ["0.1-0.2"]
 julia = "0.6-0"
 
-["0.3-0"]
+["0.3-0.5.1"]
 julia = "0.7-1"
```

Comment:
Please keep this

Reply:
@nalimilan - can you please have a look at this, I do not have much experience with package version management.

Reply:
Just add
```toml
[compat]
julia = "1"
```
to the project file and retrigger the bot. (Or `julia = "0.7, 1"` if ot works on 0.7 too.)

Reply:
it should be fixed now